### PR TITLE
docs: rebrand layer0 to edgio

### PR DESCRIPTION
**Pull request by rishi-raj-jain**

### CODE REVIEW

**Comment by rishi-raj-jain** [comment]
I don't think the pt-br guides are being used. Let me know if I shall add that as well.

Marking this ready to review now.

File: content/en/deployments/2.edgio.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+How to deploy a Nuxt app with Edgio?
+
+---
+
+Edgio supports both static and universal (SSR) Nuxt applications.
+
+[Edgio Applications](https://docs.edg.io) is an Internet-scale platform that makes it easy for teams to build, release, protect, and accelerate their web apps and APIs.
+
+For detailed instructions consult the [Edgio's Nuxt Integration Guide](https://docs.edg.io/guides/nuxt)
```
Comment (danielroe) [inline content/en/deployments/2.edgio.md:20]:
```suggestion
For detailed instructions consult [Edgio's Nuxt Integration Guide](https://docs.edg.io/guides/nuxt)
```

File: content/es/deployments/2.edgio.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+How to deploy a Nuxt app with Edgio?
+
+---
+
+Edgio supports both static and universal (SSR) Nuxt applications.
+
+[Edgio Applications](https://docs.edg.io) is an Internet-scale platform that makes it easy for teams to build, release, protect, and accelerate their web apps and APIs.
+
+For detailed instructions consult the [Edgio's Nuxt Integration Guide](https://docs.edg.io/guides/nuxt)
```
Comment (danielroe) [inline content/es/deployments/2.edgio.md:20]:
```suggestion
For detailed instructions consult [Edgio's Nuxt Integration Guide](https://docs.edg.io/guides/nuxt)
```

**Review by danielroe — APPROVED** [review]

**Comment by rishi-raj-jain** [comment]
Thanks @danielroe for the super fast work!

**Pull request merged**